### PR TITLE
Update argo docs

### PR DESCRIPTION
**Pull request by dtuite**
Preview: https://deploy-preview-567--roadie.netlify.app/docs/integrations/argocd/

Also added this page: https://deploy-preview-567--roadie.netlify.app/docs/details/setting-secrets/

### CODE REVIEW

**Comment by dtuite** [comment]
https://app.shortcut.com/larder/story/6333/update-argo-docs

File: content/docs/integrations/argocd/index.md [hunk 1]
```diff
@@ -1,62 +1,73 @@
...
+![Set Argo CD Config](./config.png)
 
-### Step 4: Add the UI elements
+## Step 4: Add the UI elements
 
-The ArgoCD plugin provides two type of UI elements. 
+The Argo CD plugin provides two type of UI elements. 
 
-The `EntityArgoCDOverviewCard` presents the current status of an application: 
+The `EntityArgo CDOverviewCard` presents the current status of an application:
```
Comment (Xantier) [inline content/docs/integrations/argocd/index.md:53]:
```suggestion
The `EntityArgoCDOverviewCard` presents the current status of an application: 
```

File: content/docs/integrations/argocd/index.md [hunk 2]
```diff
@@ -1,62 +1,73 @@
...
+The Argo CD plugin provides two type of UI elements. 
 
-The `EntityArgoCDOverviewCard` presents the current status of an application: 
+The `EntityArgo CDOverviewCard` presents the current status of an application: 
 
-![ArgoCD Overview Card](./argo-overview.png)
+![Argo CD Overview Card](./argo-overview.png)
 
-The `EntityArgoCDHistoryCard` displays the deployment history of an appliction:
+The `EntityArgo CDHistoryCard` displays the deployment history of an appliction:
```
Comment (Xantier) [inline content/docs/integrations/argocd/index.md:57]:
```suggestion
The `EntityArgoCDHistoryCard` displays the deployment history of an appliction:
```

File: content/docs/integrations/argocd/index.md [hunk 3]
```diff
@@ -1,62 +1,73 @@
...
 
-![ArgoCD History Card](./argo-history.png)
+![Argo CD History Card](./argo-history.png)
 
 Both can be [added to component dashboards](/docs/getting-started/updating-the-ui/#updating-dashboards).
 
-The `EntityArgoCDContent` displays similar information to the history card and can be [added as a tab](/docs/getting-started/updating-the-ui#updating-tabs) to component layouts.
+The `EntityArgo CDContent` displays similar information to the history card and can be [added as a tab](/docs/getting-started/updating-the-ui#updating-tabs) to component layouts.
```
Comment (Xantier) [inline content/docs/integrations/argocd/index.md:63]:
```suggestion
The `EntityArgoCDContent` displays similar information to the history card and can be [added as a tab](/docs/getting-started/updating-the-ui#updating-tabs) to component layouts.
```

**Review by Xantier — APPROVED** [review]

**Pull request merged**